ambry-server/src/main/java/com/github/ambry/server/StorageRestRequestService.java

```diff
@@ -31,9 +31,12 @@
 public class StorageRestRequestService implements RestRequestService {
   private static final Logger logger = LoggerFactory.getLogger(StorageRestRequestService.class);
   NettyServerRequestResponseChannel requestResponseChannel;
+  ServerSecurityService serverSecurityService;
 
-  public StorageRestRequestService(NettyServerRequestResponseChannel requestResponseChannel) {
+  public StorageRestRequestService(NettyServerRequestResponseChannel requestResponseChannel,
```

Comment:
Why pass in serverSecurityService here? 

BTW, this `StorageRestRequestService` will be deprecated in https://github.com/linkedin/ambry/pull/1554

Reply:
reverted the changes in this file for now as this will be deprecated #1554 and any future validation on individual stream can be done in the new class